Alien.h

```diff
@@ -1,3 +1,6 @@
+#ifndef ALIENBASEBALL_ALIEN_H_
+#define ALIENBASEBALL_ALIEN_H_
```

Comment:
Better is to use `#pragma once` unless you can't.
All modern compilers support `#pragma once` - gcc, clang, msvc - even though it's not an official part of the standards.
And it's superior because you don't have to pick an arbitrary symbol name (and hope it's unique). Plus it's one line instead of three.